wiki/Ranking_Criteria/en.md

```diff
@@ -30,6 +30,7 @@ Important to understand before using:
 ### Rules
 
 - **No two hit objects can be placed on the same tick.** This includes hit circles and the durations of sliders and spinners. osu!mania difficulties are exempt from this.
+- **No two inputs can be less than 20 milliseconds apart.** osu!mania and osu!catch difficulties are exempt from this.
```

Comment:
apparently this was [addressed in the thread](https://osu.ppy.sh/community/forums/topics/1563145?n=2) with the wording of "active input" (i.e. key down; as in active object, things you click on), but the wording here seems to miss that point. i'd prefer using "active object" here since it encompasses the same meaning, although there should probably be a stub for it first (or have the [osu!catch-related stub](https://osu.ppy.sh/wiki/en/Glossary/Active_object) expanded)

Reply:
this refers to 2 keydowns as wala says. the idea is to allow sliderend->next object timeframes to be smaller than 20ms. that said, i've changed "inputs" -> "key presses" for now, and can do what wala suggests about article expansion in the future

Reply:
I definitely prefer "key press" or "key down", since "active input" means nothing to us as developers, let alone users.